### PR TITLE
Invalid build number for candidate scraper should fail #445

### PR DESCRIPTION
**Pull request by Utkarsh1308**

### CODE REVIEW

**Comment by whimboo** [comment]
For this PR please also have a look at PR #513 where I gave a review comment which also applies here.

Also both the Appveyor and Travis CI tests fail. Please have a look and check if you can find the problem.

**Comment by Utkarsh1308** [comment]
Hey! For some reason my tox command is failing. It displays this error message "ERROR:  py27: InterpreterNotFound: python2.7"

**Comment by Utkarsh1308** [comment]
@whimboo 3 of my tests are failing without any changes - https://pastebin.com/GHzMC9Nr

**Comment by whimboo** [comment]
This probably happened due to `The 'treeherder-client==5.0.0' distribution was not found and is required by mozdownload`. It could have been a temporary problem. So please retry to run the tests.

**Comment by whimboo** [comment]
Oh, and make sure that you are on the latest master revision.

File: mozdownload/scraper.py [hunk 1]
```diff
@@ -697,14 +697,16 @@ def get_build_info(self):
         self.builds = parser.entries
         self.build_index = len(parser.entries) - 1
 
-        if self.build_number and \
-                ('build%s' % self.build_number) in self.builds:
```
Comment (whimboo) [inline mozdownload/scraper.py]: Same as for the original PR where I added this comment: https://github.com/mozilla/mozdownload/pull/513/files#r246667522 Please fix that.

**Comment by whimboo** [comment]
No reply from @Utkarsh1308 here. So closing the PR.

**Pull request closed without merging**